src/main/java/com/spotify/reaper/resources/ClusterResource.java

```diff
@@ -226,7 +181,7 @@ public Response deleteCluster(@PathParam("cluster_name") String clusterName) {
     }
     Optional<Cluster> deletedCluster = context.storage.deleteCluster(clusterName);
     if (deletedCluster.isPresent()) {
-      return Response.ok().entity(new ClusterStatus(deletedCluster.get())).build();
```

Comment:
Why are you removing the returned deleted object from the response? This makes giving in the limit parameter also strange?